mantidproject/mantid
# Scripting window remembers open files

Fixes #13082 

Scripting window will now remember the previously opened scripts and reload them after the application is restarted.
# To test:
# Test open tabs
- Start Mantid Plot
- Open the Scripting window (F3 or "View > Script Window")
- Open any couple of python scripts
- Close Mantid Plot
- Start Mantid Plot again
- Open the Scripting window
  - The previous opened scripts should be in the window already
# Test empty window
- Start Mantid Plot
- Opening the Scripting window (F3 or "View > Script Window")
- Close all the tabs in the Scripting window (right click on a tab and select "Close all")
  - The Scripting window should now be blank (no open tabs)
- Close Mantid Plot
- Start Mantid Plot again
- Open the Scripting window
- The Scripting window should contain a blank tab labelled "New script"
